docs/contributors/dev.rst

```diff
@@ -0,0 +1,75 @@
+.. Author : Gwendall PETIT (Lab-STICC - CNRS UMR 6285 / DECIDE Team)
+
+Develop 
+========
+
+Wiki documentation
+--------------------------
+
+Below are some useful links to developers documentation, that are mainly provided on the OrbisGIS GitHub_ account.
+
+.. _GitHub: https://github.com/orbisgis/orbisgis/wiki
+
+
+* `How to build OrbisGIS ?`_
+* `Debugging OrbisGIS with maven`_
+* `OrbisGIS and Nexus`_
+* `Release on Maven Central`_
+* `OrbisGIS internals`_
+* `OrbisGIS plugin system`_
+	* `Create your plugin`_
+	* `Resolve your plugin's dependencies`_
+
+
+.. _How to build OrbisGIS ?: https://github.com/orbisgis/orbisgis/wiki/2.-Building-OrbisGIS
+.. _Debugging OrbisGIS with maven: https://github.com/orbisgis/orbisgis/wiki/3.-Debugging-OrbisGIS-with-maven
+.. _OrbisGIS and Nexus: https://github.com/orbisgis/orbisgis/wiki/4.-OrbisGIS-and-Nexus
+.. _Release on Maven Central: https://github.com/orbisgis/orbisgis/wiki/5.-Release-on-Maven-Central
+.. _OrbisGIS internals: https://github.com/orbisgis/orbisgis/wiki/7.-OrbisGIS-internals
+.. _OrbisGIS plugin system: https://github.com/orbisgis/orbisgis/wiki/8.-OrbisGIS-plugin-system
+.. _Create your plugin: https://github.com/orbisgis/orbisgis/wiki/8.1.-Create-your-plugin
+.. _Resolve your plugin's dependencies: https://github.com/orbisgis/orbisgis/wiki/8.2.-Resolve-your-plugin's-dependencies
+ 
+
+Downloads and links
+-----------------------------------
+
+* Releases and pre-releases versions can be found `HERE`_
+* `Daily build version`_ (for test purpose)
+* Continuous Integration
+	* `Jenkins server`_
+	* `Travis`_
+
+.. _HERE: https://github.com/orbisgis/orbisgis/releases
+.. _Daily build version: http://jenkins.orbisgis.org/job/orbisgis/lastSuccessfulBuild/artifact/orbisgis-dist/target/orbisgis-bin.zip
+.. _Jenkins server: http://jenkins.orbisgis.org/
+.. _Travis: https://travis-ci.org/orbisgis/orbisgis
+
+
+Contribute
+-----------------------
+
+Developers are cordially invited to contribute to OrbisGIS via:
+
+* Issues: detect a bug, ask for enhancement ? `Create an issue`_ on GitHub
+* `Pull Request`_ (PR): want to contribute ? Please submit your PR and we will analyze it carefully.
+
+.. _Create an issue: https://github.com/orbisgis/orbisgis/issues/new
+.. _Pull Request: https://github.com/orbisgis/orbisgis/pulls
+
+
+Additional informations
+-----------------------------
+
+* Made by a `team`_ from `CNRS`_
+* Registered date : 22/12/2007
+* Langage of development : 100% Java
+* Licence : `GPL v3`_
+* Supported OS : Linux, Windows and Mac OS
+* Installation requirement : at least Java 7
+
+.. _team: https://github.com/orbisgis/orbisgis/wiki/1.-Team
+.. _CNRS: http://www.cnrs.fr/
+.. _GPL v3: https://www.gnu.org/licenses/quick-guide-gplv3.en.html
```

Comment:
I propose this plan

Source code
GITHUB

Tools for developers
JENKINS
TRAVIS

Releases
Stable
Last

Documentation